### PR TITLE
Mobil menu pull request

### PR DESCRIPTION
**Pull request by codehass**
In this pull request : 
- Add HTML-CSS-js linters
- Add images for the menu
- Add HTML code
- Style the HTML code
- Make required changes.

Thank you for reviewing my pull request 👍 .

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function showMenu() {
+  document.querySelector('.nav-desk').classList.toggle('active');
+  document.querySelector('.nav-mobil-img').classList.toggle('active');
+  document.querySelector('.nav-mobil-img-2').classList.toggle('active');
+  document.querySelector('.body').classList.toggle('active');
+  document.querySelector('.menu').classList.toggle('hide');
+  document.querySelector('.close').classList.toggle('show');
+}
+document.querySelector('.nav-desk').addEventListener('click', showMenu());
```
Comment (omarsalem7) [inline script.js]: - [ ] Kindly make changes from [here](https://www.loom.com/share/dd0b3a7d3e554da9b875e0f7d229ffc8) in the loom video.
- [ ] Also make sure when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @codehass 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+document.querySelector('.menu-icon').addEventListener('click', () => {
+  document.querySelector('.nav-desk').classList.toggle('active');
```
Comment (Gopxfs) [inline script.js:2]: As requested by the previous reviewer, please make sure that your links redirect the user to the corresponding page 😉.
Kindly notice that currently, you have plain `li` items as `nav` links:
image

To achieve the desired result, kindly add an `a` tag with the corresponding `href`.
You could have something like this:

```HTML
<ul>
    <li><a href="#">Hello</a></li>
    <li><a href="#portfolio">Portfolio</a></li>
    <li><a href="#about">About</a></li>
    <li><a href="#contact">Contact</a></li>
</ul>
```

If you are interested in learning more about `a` tags, check [this link](https://www.w3schools.com/tags/tag_a.asp) 😄.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @codehass 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -23,26 +23,39 @@
...
           <ul>
-            <li>Hello</li>
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+            <li class="menu-sel"><a href="#hero">Hello</a></li>
+            <li class="menu-sel"><a href="#portfolio">Portfolio</a></li>
+            <li class="menu-sel"><a href="#about">About</a></li>
+            <li class="menu-sel"><a href="#contact">Contact</a></li>
           </ul>
```
Comment (tufoinnkuo10) [inline index.html:51]: - [optional] kindly maintain the figma color on the menu links ...`#000000;`

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @codehass 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- menu works as expected
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**